### ISSUE TITLE
News Detail pages have <nav> element in DOM but no visual sidenav

### ISSUE THREAD

**Issue by laflannery** [department-of-veterans-affairs/va.gov-cms#20174]
**Note:** I swear I already had this ticketed because I've known about this but when I went to pull it into the Next-build QA I couldn't find it so I'm making it again. Just noting that because there might be a dup somewhere

## Description
On the News Release detail pages ([example](https://www.va.gov/boston-health-care/news-releases/aerosols-may-contribute-to-hospital-transmission-of-covid-19/)), there is an empty `<nav>` element in the DOM that is an accessibility issue. This creates a landmark (which often folks navigate by) and since there is nothing there, this could cause confusion. As there is never an actual navigation on these pages anyway, we should remove this element from the DOM

![Image](https://github.com/user-attachments/assets/4d55e255-c8ef-45b5-9028-101e90aaefce)

## Acceptance criteria

- [ ] There should be no secondary `<nav>` element in the DOM because these page templates do not have a sidebar navigation
- [ ] Accessibility review

**Comment by laflannery** [comment]
Turns out this was actually removed and correct in next so I'm going to close this (and if the dupe is ever found we can close that too!)
